maintainer/conda/environment.yml

```
@@ -25,6 +25,7 @@ dependencies:
   - scipy
   - pip
   - sphinx <7.0
+  - transformations>=2018
```

```suggestion
  - transformations>=2018
```
What is the reason for this pin? As far as I can tell, 2018 is the first version in PyPI and conda-forge only has 2022.

---

followed co-pilot's suggestion to add minimum version ... but will revert